fragments/filter/README.md

```diff
@@ -507,11 +491,12 @@ OGC API Features filters only operate against a single collection already.
 #### Example 2: GET with cql-text
 
 ```javascript
-GET /search?filter=collection = 'landsat8_l1tp' 
+GET /search?filter=collection = 'landsat8_l1tp'
   AND gsd <= 30
-  AND eo:cloud_cover <= 10 
-  AND datetime ANYINTERACTS 2021-04-08T04:39:23Z/2021-05-07T12:27:57Z 
-  AND INTERSECTS(geometry, POLYGON((43.5845 -79.5442, 43.6079 -79.4893, 43.5677 -79.4632, 43.6129 -79.3925, 43.6223 -79.3238, 43.6576 -79.3163, 43.7945 -79.1178, 43.8144 -79.1542, 43.8555 -79.1714, 43.7509 -79.6390, 43.5845 -79.5442)) 
+  AND eo:cloud_cover <= 10
+  AND datetime >= "2021-04-08T04:39:23Z"
```

Comment:
Wouldn't this usually be covered by a BETWEEN?

Reply:
It would, but BETWEEN is in the Advanced Comparison Operators conformance class, and I wanted these first examples to only use the Basic operators.